src/sentry/static/sentry/app/components/assigneeSelector.tsx

```diff
@@ -373,16 +380,19 @@ const IconContainer = styled('div')`
   flex-shrink: 0;
 `;
 
-const MenuItemWrapper = styled(({py, ...props}) => <div {...props} />)`
+const MenuItemWrapper = styled('div')<{
+  px?: number;
```

Comment:
@leedongwei this `px` prop seems like it's used at all; and we can safely remove it .